Meadowlark-js/tests/e2e/scenarios/SecurityValidation.test.ts

```diff
@@ -0,0 +1,372 @@
+    clientName: 'Vendor 2',
+    roles: ['vendor'],
+  };
+  let host1Data: Credentials = {
+    clientName: 'host 1',
+    roles: ['host'],
+  };
+
+  let vendor1DataAccessToken = '';
+  let vendor2DataAccessToken = '';
+  let host1DataAccessToken = '';
+
+  let contentClassDescriptorLocation = '';
+
+  let educationContentLocation = '';
+  const educationContentEndpoint = 'EducationContents';
+  const educationContentBody = {
+    contentIdentifier: '933zsd4350',
+    namespace: '43210',
+    shortDescription: 'abc',
+    contentClassDescriptor: 'uri://ed-fi.org/ContentClassDescriptor#Presentation',
+    learningResourceMetadataURI: '21430',
+  };
+
+  beforeAll(async () => {
+    vendor1Data = await createClient(vendor1Data);
+    vendor2Data = await createClient(vendor2Data);
+    host1Data = await createClient(host1Data);
+
+    // Vendor 1
+    await baseURLRequest()
+      .post('/oauth/token')
+      .send({
+        grant_type: 'client_credentials',
+        client_id: vendor1Data.key,
+        client_secret: vendor1Data.secret,
+      })
+      .expect(200)
+      .then((response) => {
+        vendor1DataAccessToken = response.body.access_token;
+      });
+
+    // Vendor 2
+    await baseURLRequest()
+      .post('/oauth/token')
+      .send({
+        grant_type: 'client_credentials',
+        client_id: vendor2Data.key,
+        client_secret: vendor2Data.secret,
+      })
+      .expect(200)
+      .then((response) => {
+        vendor2DataAccessToken = response.body.access_token;
+      });
+
+    // Host 1
+    await baseURLRequest()
+      .post('/oauth/token')
+      .send({
+        grant_type: 'client_credentials',
+        client_id: host1Data.key,
+        client_secret: host1Data.secret,
+      })
+      .expect(200)
+      .then((response) => {
+        host1DataAccessToken = response.body.access_token;
+      });
+
+    contentClassDescriptorLocation = await createResource({
+      endpoint: 'contentClassDescriptors',
+      body: {
+        codeValue: 'Presentation',
+        shortDescription: 'Presentation',
+        description: 'Presentation',
+        namespace: 'uri://ed-fi.org/ContentClassDescriptor',
+      },
+      accessToken: vendor1DataAccessToken,
+    });
+
+    expect(contentClassDescriptorLocation).toBeTruthy();
+  });
+
+  afterAll(async () => {
+    await deleteResourceByLocation(contentClassDescriptorLocation);
+  });
+
+  // GET
+  describe('given that Vendor 1 creates an educationContent', () => {
+    beforeAll(async () => {
+      await baseURLRequest()
+        .post(`/v3.3b/ed-fi/${educationContentEndpoint}`)
+        .send(educationContentBody)
+        .auth(vendor1DataAccessToken, { type: 'bearer' })
+        .expect(201)
+        .then((response) => {
+          expect(response).not.toBe('');
+          educationContentLocation = response.headers.location;
+        });
+
+      expect(educationContentLocation).toBeTruthy();
+    });
+
+    it('Vendor 1 can get it', async () => {
```

Comment:
Back to "given, when, then" - here's a case where it does make sense to keep "when" in between "given" and "then" (it) - or maybe just omit it entirely if it isn't helpful.

You could have something like this:

```
describe('when the same vendor requests the resource'
    it('should receive the resource')

describe('when a different vendor requests the resource' 
   it('should be denied')

describe('when a host account requests the resource'
    it('should receive the resource')
```

When written this way, it also reinforces the requirements that are being tested.